hrjobcontract/js/test/controllers/contract_test.js

```diff
@@ -1,22 +1,57 @@
 define([
+    'common/moment',
     'common/angularMocks',
-    'job-contract/app'
+    'job-contract/app',
+    'job-contract/controllers/contract'
 ], function () {
     'use strict';
 
     describe('ContractCtrl', function () {
-        var ctrl;
+        var ctrl, $rootScope, $scope, $controller;
 
         beforeEach(module('hrjc'));
-        beforeEach(inject(function ($controller, $rootScope) {
-            var $scope = $rootScope.$new();
-            $scope.contract = { id: '1' };
+        beforeEach(inject(function (_$controller_, _$rootScope_) {
+
+          $controller = _$controller_;
+          $rootScope = _$rootScope_;
+
+          $scope = $rootScope.$new();
+
+          $scope.contract = {
+            id: '1',
+            contact_id: "84",
+            deleted: "0",
+            is_current: "0",
+            is_primary: "1"
+          };
+
+          $scope.$parent.contract = {
+            id: '1',
+            contact_id: "84",
+            deleted: "0",
+            is_current: "0",
+            is_primary: "1"
+          };
+
+          $scope.$parent.contractCurrent = [];
+          $scope.$parent.contractPast = [];
+
+          ctrl = $controller('ContractCtrl', {
+            $scope: $scope
+          });
 
-            ctrl = $controller('ContractCtrl', { $scope: $scope });
         }));
 
-        it('example', function () {
-            expect(true).toBe(true);
+        it('should make current contract', function () {
+          var d = new Date().toISOString().split("T")[0];
+          $scope.updateContractList(d);
```

Comment:
An `it` block should contain, ideally, only one `expect` call and nothing else. Any setup needed for testing the expectation should go in a `beforeEach` block
```js
describe('lorem ipsum', function () { // <-- Dummy name because i don't know yet what you are testing here
  describe('when the date is ??, function () { // <-- I don't know, fill in the blank
    beforeEach(function () {
      $scope.updateContractList(/* some date */);
    });

    it('makes current contract', function () {
      expect($scope.$parent.contract.is_current).toBe("1");
    });
  });

  describe('when the date is ??, function () { // <-- I don't know, fill in the blank
    beforeEach(function () {
      $scope.updateContractList(/* some date */);
    });

    it('makes past contract', function () {
      expect($scope.$parent.contract.is_current).toBe("1");
    });
  });
});
```